src/page-response-impact/rtStatistics.tsx

```diff
@@ -0,0 +1,42 @@
+import { RtRecord } from "../infection-model/rt";
+
+export function numFacilitiesWithRtLessThan1(
+  facilitiesRtRecords: RtRecord[][],
+) {
+  const latestRtValues = facilitiesRtRecords.map((rtRecords) => {
+    // Note: This assumes the first in the array is the latest because of the
+    // sorting on line 104 in infection-model/rt.tsx
+    return rtRecords[0].value;
+  });
+  return latestRtValues.filter((v) => v < 1).length;
+}
+
+export function averageRtReductionAcrossFacilities(
+  facilitiesRtRecords: RtRecord[][],
+) {
+  const numFacilities = facilitiesRtRecords.length;
+  const rtDifferences: number[] = facilitiesRtRecords.map(
+    (rtRecords: RtRecord[]) => {
+      const latestRtValue = rtRecords[0].value;
+      const oldestRtValue = rtRecords[rtRecords.length - 1].value;
```

Comment:
seeing how much we need to access these (not just here but in other Rt-related features too) is making me think we ought to have helper functions like `getOldestRt`, `getNewestRt`, or whatever, to encapsulate that logic and make it easier to change

Reply:
That sounds good, also I can write those to check the data instead of relying on the order of the array.